flow/scenarios/loop.py

```diff
@@ -196,7 +195,20 @@ def specify_edge_starts(self):
         """See parent class."""
         edgelen = self.net_params.additional_params["length"] / 4
 
-        edgestarts = [("bottom", 0), ("right", edgelen), ("top", 2 * edgelen),
-                      ("left", 3 * edgelen)]
+        edgestarts = [("bottom", 0),
```

Comment:
Magic numbers! No magic numbers without at least a comment explaining why they're there (or alternately, a name that explains why they're there)